### ISSUE TITLE
Graphql error! Invalid API Key Provided

### ISSUE THREAD

**Issue by osamajandali** [withspectrum/spectrum#2716]
I have setup everything correctly on my localhost and everything works just fine except when trying to access home and do a new thread I get this error log. 
```
  shared:middlewares:logging requesting /api +8s
  shared:middlewares:logging requesting /api +4ms
  api:utils:error-formatter ---GraphQL Error--- +902ms
  api:utils:error-formatter Error: Invalid API Key provided: *****ined
  api:utils:error-formatter     at Constructor._Error (/Users/osamajandali/Desktop/Work/Development/Github/spectrum/node_modules/stripe/lib/Error.js:12:17)
  api:utils:error-formatter     at Constructor (/Users/osamajandali/Desktop/Work/Development/Github/spectrum/node_modules/stripe/lib/utils.js:102:13)
  api:utils:error-formatter     at new Constructor (/Users/osamajandali/Desktop/Work/Development/Github/spectrum/node_modules/stripe/lib/utils.js:102:13)
  api:utils:error-formatter     at IncomingMessage.<anonymous> (/Users/osamajandali/Desktop/Work/Development/Github/spectrum/node_modules/stripe/lib/StripeResource.js:166:21)
  api:utils:error-formatter     at emitNone (events.js:111:20)
  api:utils:error-formatter     at IncomingMessage.emit (events.js:208:7)
  api:utils:error-formatter     at endReadableNT (_stream_readable.js:1055:12)
  api:utils:error-formatter     at _combinedTickCallback (internal/process/next_tick.js:138:11)
  api:utils:error-formatter     at process._tickDomainCallback (internal/process/next_tick.js:218:9)
  api:utils:error-formatter From previous event:
  api:utils:error-formatter     at Constructor.create (/Users/osamajandali/Desktop/Work/Development/Github/spectrum/node_modules/stripe/lib/StripeMethod.js:36:29)
  api:utils:error-formatter     at createCustomer (/Users/osamajandali/Desktop/Work/Development/Github/spectrum/build-api/webpack:/shared/stripe/utils.js:156:43)
  api:utils:error-formatter     at Object.jobPreflight (/Users/osamajandali/Desktop/Work/Development/Github/spectrum/build-api/webpack:/shared/stripe/utils.js:367:35)
  api:utils:error-formatter     at <anonymous> +1ms
  api:utils:error-formatter -------------------
  api:utils:error-formatter  +0ms
Error: Invalid API Key provided: *****ined
    at Constructor._Error (/Users/osamajandali/Desktop/Work/Development/Github/spectrum/node_modules/stripe/lib/Error.js:12:17)
    at Constructor (/Users/osamajandali/Desktop/Work/Development/Github/spectrum/node_modules/stripe/lib/utils.js:102:13)
    at new Constructor (/Users/osamajandali/Desktop/Work/Development/Github/spectrum/node_modules/stripe/lib/utils.js:102:13)
    at IncomingMessage.<anonymous> (/Users/osamajandali/Desktop/Work/Development/Github/spectrum/node_modules/stripe/lib/StripeResource.js:166:21)
    at emitNone (events.js:111:20)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1055:12)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickDomainCallback (internal/process/next_tick.js:218:9)
From previous event:
    at Constructor.create (/Users/osamajandali/Desktop/Work/Development/Github/spectrum/node_modules/stripe/lib/StripeMethod.js:36:29)
    at createCustomer (/Users/osamajandali/Desktop/Work/Development/Github/spectrum/build-api/webpack:/shared/stripe/utils.js:156:43)
    at Object.jobPreflight (/Users/osamajandali/Desktop/Work/Development/Github/spectrum/build-api/webpack:/shared/stripe/utils.js:367:35)
    at <anonymous>

```

**Comment by brianlovin** [comment]
Should get resolved with #2715

**Comment by osamajandali** [comment]
@brianlovin Thanks for quick response 👍
